ui/components/multichain/asset-picker-amount/asset-picker-modal/asset-picker-modal.tsx

```diff
@@ -547,8 +570,10 @@ export function AssetPickerModal({
       <ModalContent modalDialogProps={{ padding: 0 }}>
         <ModalHeader
           onClose={() => {
-            setSearchQuery('');
+            debouncedSetSearchQuery.cancel();
+            abortControllerRef.current?.abort();
             onClose();
+            // setSearchQuery('');
```

Comment:
Was this a mistake? I'm guessing that we want this `onClose` handler to do the same as the `onClose` handler for the overall modal.